core/client/app/controllers/settings/tags.js

```diff
@@ -31,13 +27,7 @@ export default Ember.Controller.extend({
     }),
 
     actions: {
-        enteredMobile: function () {
-            this.set('isMobile', true);
-        },
-
         leftMobile: function () {
```

Comment:
The `isMobile` property is used in the [gh-tag-settings-form](https://github.com/TryGhost/Ghost/blob/master/core/client/app/templates/components/gh-tag-settings-form.hbs#L3) component template so that we can change the header styles and add navigation back to the tags list from the tag settings screen.

Perhaps it would make more sense to use the `mediaQueries` service inside the `tags` controller so that `isMobile` is easier to pass in to the various components? I believe having the checks in the container component is a hangover from a previous attempt at watching the size of the container rather than overall window size.